src/test/java/be/seeseemelk/mockbukkit/scheduler/BukkitSchedulerMockTest.java

```diff
@@ -179,4 +180,86 @@ void runTaskTimerAsynchronously_TaskExecutedOnSeperateThread() throws Interrupte
 		assertFalse(scheduler.isQueued(testTask.getTaskId()));
 		assertEquals(2, count.get());
 	}
+
+	@Test
+	public void getPendingTasks_Sync()
+	{
+		assertTrue(scheduler.getPendingTasks().isEmpty());
+		int amountTasks = 20;
+		AtomicInteger count = new AtomicInteger(amountTasks);
+		Runnable callback = count::decrementAndGet;
+		for (int i = 0; i < amountTasks; i++)
+		{
+			scheduler.runTaskLater(null, callback, 2L+(i%5));
+		}
+		while (count.get()>0)
+		{
+			assertEquals(count.get(), scheduler.getPendingTasks().size());
+			scheduler.performOneTick();
+		}
+		assertEquals(count.get(), scheduler.getPendingTasks().size());
```

Comment:
This could be simplified a little as you're making sure that `count.get()` is zero anyway.

```suggestion
		assertEquals(0, scheduler.getPendingTasks().size());
```